src/helpers/subgraph/ProtocolMetricsHelper.ts

```diff
@@ -0,0 +1,157 @@
+import get from "get-value";
+import { CSSProperties } from "react";
+import { ChartType } from "src/components/Chart/Constants";
+import { getFloat } from "src/helpers/NumberHelper";
+
+export const objectHasProperty = (object: unknown, property: string): object is Record<string, unknown> => {
+  return typeof object === "object" && object !== null && property in object;
+};
+
+/**
+ * As the subgraph data source takes a long time to re-index, we alter
+ * the display name of certain tokens here.
+ *
+ * This should be used sparingly, and the updated display names should be integrated
+ * into the subgraph ASAP.
+ */
```

Comment:
You have the world's greatest jsdocs. All this code is so clean.

Reply:
🙏